### ISSUE TITLE
Question about training.

### ISSUE THREAD

**Issue by HenryHZY** [zhengrongz/AoTD#3]
@zhengrongz Hi, thanks for your great project. 

As mentioned in the paper's Sec. 3.4. Step-by-step Distillation, LLaVA-NeXT-Video-AoTD is trained with the loss of `L = L_label+ λL_rationale`.

- Q1: Have you trained L_label and L_rationale separately? (e.g., use two training samples following the data format in https://github.com/zhengrongz/AoTD/tree/main/distillation) 

- Q2: Where can we find the checkpoints of LLaVA-NeXT-Video-AoTD? 

- Q3: Given a question, does LLaVA-NeXT-Video-AoTD directly answer this question (like Figure 4)? Or do we need to generate CoT and then convert it into def execute_command(), so as to call specialized models?

By the way, do you have a plan to release the generated CoT for training datasets? Thank you.

**Comment by HenryHZY** [comment]
Got it. Thanks for your quick response!
